frontend/src/api/Socket.ts

```diff
@@ -71,14 +72,26 @@ export const subscribe = (subscribeUrl: string,
     }
   });
 
+/**
+ * Subscribe the user via socket.
+ * @returns void Promise, reject if socket is not subscribed to that URL.
+ */
+export const send = (sendUrl: string, headers?: {}, body?: string):
+  Promise<void> => new Promise<void>((resolve, reject) => {
+    if (connected) {
```

Comment:
This is true, good point -- #88, [here](https://github.com/rocketden/main/pull/88/files#diff-b449d88672253d971a98287d30a0be8f57baf0c77f16cfafc22e773e253962e3L38-R35). I made the change indicated below, only on this `send` function, because the rest of the changes will be completed by the #88 merge. Change in this PR:

OLD: `if (connected) {`
NEW: `if (stompClient && stompClient.connected) {`